src/common/queries/scheduledOrders.ts

```diff
@@ -0,0 +1,98 @@
+import { Account } from '../../account'
+import { Address } from 'viem'
+import { GRAPHQL_API_URL } from '../constants'
+
+const query = `
+    query ($smartAccount: String) {
+      scheduledOrdersExecutionAddedQueries (where:{ smartAccount: $smartAccount}) {
+        id
+        jobId
+        smartAccount
+        executeInterval
+        numberOfExecutions
+        numberOfExecutionsCompleted
+        startDate
+        isEnabled
+      }
+    }
+  `
+
+export const getScheduledOrders = async ({
+  account,
+}: {
+  account: Account
+}): Promise<Address[]> => {
+  const variables = {
+    smartAccount: account.address,
+  }
+
+  const response = await fetch(GRAPHQL_API_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      query,
+      variables,
+    }),
+  })
+
+  const responseBody = await response.json()
+
+  console.log('responseBody', responseBody)
+
+  if (response.ok) {
+    console.log('responseBody.data', responseBody.data)
+    return responseBody.data.scheduledOrdersExecutionAddedQueries // responseBody.data.moduleQueries.map((module: any) => module.module)
+  } else {
+    throw new Error(
+      `Error: ${responseBody.errors.map((error: any) => error.message).join(', ')}`,
+    )
+  }
+}
+
+const queryById = `
+    query ($jobId: String) {
+      scheduledOrdersExecutionAddedQueries (where:{ jobId: $jobId}) {
```

Comment:
this also requires the smartaccount right?

Reply:
Yep, Fixed